packages/zcli-themes/src/commands/themes/update.ts

```diff
@@ -0,0 +1,46 @@
+import { Command, Flags, CliUx } from '@oclif/core'
+import * as path from 'path'
+import * as chalk from 'chalk'
+import createThemeUpdateJob from '../../lib/createThemeUpdateJob'
+import createThemePackage from '../../lib/createThemePackage'
+import uploadThemePackage from '../../lib/uploadThemePackage'
+import pollJobStatus from '../../lib/pollJobStatus'
+
+export default class Update extends Command {
+  static description = 'import a theme'
+
+  static flags = {
+    themeId: Flags.string({ description: 'The id of the theme to update' }),
+    replaceSettings: Flags.boolean({ default: false, description: 'Whether or not to replace the current theme settings' })
+  }
+
+  static args = [
+    { name: 'themeDirectory', required: true, default: '.' }
+  ]
+
+  static examples = [
+    '$ zcli themes:import ./copenhagen_theme'
+  ]
+
+  static strict = false
+
+  async run () {
+    let { flags: { themeId, replaceSettings }, argv: [themeDirectory] } = await this.parse(Update)
+    const themePath = path.resolve(themeDirectory)
+
+    themeId = themeId || await CliUx.ux.prompt('Theme ID')
```

Comment:
nit: for the `import` command we are using `inquirer` to [prompt the brand Id](https://github.com/zendesk/zcli/blob/master/packages/zcli-themes/src/lib/getBrandId.ts#L16). Should they be consistent?

Reply:
Updating does not require a `brandId` as the theme is already imported in a brand so we already have a record.
So we just need the `themeId` which we also prompt for. We don't need the `inquirer` here because there's no options to select from so `CliUx.ux.prompt` is enough.

Reply:
Ahhh, gotcha! Thanks for explaining!